sdk/azidentity/azidentity.go

```diff
@@ -18,19 +18,23 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/errorinfo"
 )
 
+// The base URL for Azure Active Directory
```

Comment:
The idiom for doc comments is the must start with the thing being doc'ed, e.g. `// AzureAuthorityHost is the base URL...`